examples/usecases/transformers-next-item-prediction.ipynb

```diff
@@ -87,7 +87,7 @@
      "name": "stderr",
```

Comment:
<p>May be mention above that "... the first 4 rows of our raw validation data".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/1045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
<p>this is raw validation data, I assume? if yes, do you mind adding <code>raw</code>  just to emphasize that data is not processed. </p>